custom_components/localtuya/number.py

```diff
@@ -0,0 +1,116 @@
+"""Platform to locally control Tuya-based switch devices."""
```

Comment:
Is the "switch" terminology still valid here?

Reply:
Can't say for current version of HA. Using same version.